src/ml/train/focus_classifiers.py

```diff
@@ -82,39 +84,21 @@ def get_eeg_channels (board_id):
     print ('channels to use: %s' % str (eeg_channels))
     return eeg_channels
 
-def write_model(intercept,coefs,model_type):
-    # we prepare dataset in C++ code in compile time, need to generate header for it
-    coefficients_string = '%s' % (','.join([str(x) for x in coefs[0]]))
-    file_content = '''
-#pragma once
-
-
-// clang-format off
-
-const double %s_coefficients[10] = {%s};
-double %s_intercept = %lf ;
-
-// clang-format on
-''' % (model_type,coefficients_string, model_type,intercept)
-    file_name = f'{model_type}_model.h'
-    file_path = os.path.join (os.path.dirname (os.path.realpath (__file__)), '..', 'inc', file_name)
-    with open(file_path, 'w') as f:
-        f.write (file_content)
-
-def train_LDA(data):
-    model = LinearDiscriminantAnalysis()
-    print('#### Linear Discriminant Analysis ####')
+def train_lda (data):
+    model = LinearDiscriminantAnalysis ()
+    print ('#### Linear Discriminant Analysis ####')
     scores = cross_val_score (model, data[0], data[1], cv = 5, scoring = 'f1_macro', n_jobs = 8)
     print ('f1 macro %s' % str (scores))
     scores = cross_val_score (model, data[0], data[1], cv = 5, scoring = 'precision_macro', n_jobs = 8)
     print ('precision macro %s' % str (scores))
     scores = cross_val_score (model, data[0], data[1], cv = 5, scoring = 'recall_macro', n_jobs = 8)
-    print('recall macro %s' % str(scores))
-    model.fit(data[0], data[1])
-    write_model(model.intercept_,model.coef_,'lda')
+    print ('recall macro %s' % str (scores))
+    model.fit (data[0], data[1])
+    write_model (model.intercept_, model.coef_, 'lda')
 
 def train_regression (data):
-    model = LogisticRegression (class_weight = 'balanced', solver = 'liblinear', max_iter = 3000)
+    model = LogisticRegression (class_weight = 'balanced', solver = 'liblinear',
+        max_iter = 4000, penalty = 'l2', random_state = 1)
```

Comment:
Isn't the l2 penalty the default penalty.

Reply:
It is